packages/core-components/src/components/table/table-header/table-header.scss

```diff
@@ -11,6 +12,23 @@
   font-weight: bold;
   border-top: 1px solid var(--b2b-color-grey-200);
   border-bottom: 2px solid var(--b2b-color-grey-300);
+  overflow: hidden;
+  min-width: var(--b2b-size-200) !important;
```

Comment:
this occasionally breaks the column layout. Also please avoid !important where possible.